app/src/main/java/com/duckduckgo/app/browser/DuckDuckGoWebView.kt

```diff
@@ -439,23 +451,28 @@ class DuckDuckGoWebView : WebView, NestedScrollingChild3 {
         false
     }
 
-    @SuppressLint("RequiresFeature", "RemoveWebMessageListenerUsage")
-    suspend fun safeRemoveWebMessageListener(
-        webViewCapabilityChecker: WebViewCapabilityChecker,
-        jsObjectName: String,
-    ): Boolean = runCatching {
-        if (webViewCapabilityChecker.isSupported(WebViewCapability.WebMessageListener) && !isDestroyed) {
-            WebViewCompat.removeWebMessageListener(
-                this,
-                jsObjectName,
-            )
-            true
-        } else {
-            false
+    @SuppressLint("RequiresFeature")
+    suspend fun safeAddDocumentStartJavaScript(
```

Comment:
Do you want to add a deprecated here or something so people don't use this and instead the go through the plugin?